munezeromicha/My-BRAND-Michel
# Ft UI mybrand

## Title:
Add hamburger menu, form validation, and sliding cards to landing page

## Description:
This pull request introduces several enhancements to the landing page:

### 1. Hamburger Menu:
- Added a hamburger menu for improved navigation.
- The menu is accessible and provides easy access to important sections of the website.

### 2. Form Validation:
- Implemented form validation to enhance user experience and ensure data integrity.
- Users are prompted with informative error messages when submitting invalid input.

### 3. Sliding Cards:
- Incorporated sliding cards feature to showcase content dynamically.
- Users can now interact with cards that slide horizontally, providing a visually engaging experience.

### Testing:
To test the new features:
1. Verify that the hamburger menu appears and functions correctly on various devices and screen sizes.
2. Test form validation by submitting both valid and invalid input. Ensure appropriate error messages are displayed.
3. Interact with the sliding cards to confirm smooth animation and accurate content display.
